Atari-Game-Player/steps/train.py

```diff
@@ -0,0 +1,183 @@
+from .configs import PreTrainingConfigs
+from dqn.model import *

```

Comment:
Lets never use the `import *` thing :-) Import explicitly always, its best practice